projects/vdk-core/src/vdk/internal/builtin_plugins/run/cli_run.py

```diff
@@ -141,24 +161,8 @@ def create_and_run_data_job(
         execution_result = None
         try:
             execution_result = job.run(args)
-            # On some platforms, if the size of a string is too large, the
-            # logging module starts throwing OSError: [Errno 40] Message too long,
-            # so it is safer if we split large strings into smaller chunks.
-            string_exec_result = str(execution_result)
-            if len(string_exec_result) > 5000:
-                temp_exec_result = json.loads(string_exec_result)
-                steps = temp_exec_result.pop("steps_list")
-
-                log.info(
-                    f"Data Job execution summary: {json.dumps(temp_exec_result, indent=2)}"
-                )
-
-                chunks = math.ceil(len(string_exec_result) / 5000)
-                for i in self.__split_into_chunks(exec_steps=steps, chunks=chunks):
-                    log.info(f"Execution Steps: {json.dumps(i, indent=2)}")
-
-            else:
-                log.info(f"Data Job execution summary: {execution_result}")
+            if context.configuration.get_value("LOG_EXECUTION_RESULT"):
```

Comment:
Where is the option "LOG_EXECUTION_RESULT" defined ?

There should be in vdk_configure hook definition : 
```
def vdk_configure(self, config_builder: ConfigurationBuilder)
    config_builder.add(....
```

Reply:
We split the PR in two, as initially suggested by @dakodakov

https://github.com/vmware/versatile-data-kit/pull/2850/files

Reply:
so how do you know if a job completed successfully if this is disabled by default?

Reply:
A problem we have is that config is not super easy to discover. so ppl might not know about this.